orbit/cmd/desktop/desktop.go

```diff
@@ -23,6 +26,35 @@ import (
 // version is set at compile time via -ldflags
 var version = "unknown"
 
+func setupRunners() {
+	var runnerGroup run.Group
+
+	// Install a signal handler runner
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	var execFn func() error
+	var interrFn func(error)
+
+	if runtime.GOOS == "windows" {
+		execFn, interrFn = run.SignalHandler(ctx, os.Interrupt, os.Kill)
+	} else {
+		execFn, interrFn = run.SignalHandler(ctx, os.Interrupt, os.Kill, syscall.SIGTERM)
```

Comment:
But when receiving the signal it doesn't exit the app, it just clears the tray icon, right? Shouldn't we terminate the whole Fleet Desktop app too?

Reply:
I'm afraid no. `systray.Quit()` will terminate the whole app. So this needs to be called only when trapping the os terminate signals